pkg/api/controller_test.go

```diff
@@ -390,13 +390,15 @@ func TestController_LogCommitsPredefinedData(t *testing.T) {
 	testutil.Must(t, err)
 	const prefix = "foo/bar"
 	const totalCommits = 10
+	commitsMap := map[string]*catalog.CommitLog{}
```

Comment:
This `map` takes a key which is always `commit<INDEX>`, and all indices are in sequence.  Why not just use a slice?

```suggestion
	commits := make([]*catalog.CommitLog, totalCommits)
```